content/reference/rest-api/templates/delete.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Templates: delete'
+weight: 10
+
+menu:
+  main:
+    name: 'delete'
+    identifier: 'rest-api-templates-delete'
+    parent: 'rest-api-templates'
+    pre: 'Permanently deletes a template by ID.'
+---
+
+Permanently deletes a template and all of its versions by ID.
+
+# Request
+
+## HTTP Request
+
+```
+DELETE https://cawemo.com/api/v1/templates/:id
```

Comment:
I could imagine that we start without this endpoint, to keep things smaller in the first iteration.